util/math128.h

```diff
@@ -220,4 +220,66 @@ inline Unsigned128 DecodeFixed128(const char* ptr) {
   return (rv << 64) | DecodeFixed64(ptr);
 }
 
+// A version of EncodeFixed* for generic algorithms. Likely to be used
+// with Unsigned128, so lives here for now.
+template <typename T>
+inline void EncodeFixedGeneric(char* /*dst*/, T /*value*/) {
```

Comment:
Should we just name it `EncodeFixed()`, `DecodeFixed()`?
`FastRangeGeneric()` makes me feel it should not be used directly.

Reply:
In my opinion it should only be used for generic algorithms. Otherwise, we should be explicit about 32 vs. 64 etc. because we don't want to accidentally change some serialization format based on just changing the type of a variable. (Unless it's obviously going to do so, e.g. Thrift, but that's not normal in RocksDB.)